examples/mugen/retrieval/s3d.py

```diff
@@ -111,262 +111,42 @@ def forward(self, x):
         return x
 
 
-class Mixed3b(nn.Module):
-    def __init__(self):
-        super(Mixed3b, self).__init__()
-
-        self.branch0 = nn.Sequential(
-            BasicConv3d(192, 64, kernel_size=1, stride=1),
-        )
-        self.branch1 = nn.Sequential(
-            BasicConv3d(192, 96, kernel_size=1, stride=1),
-            SepConv3d(96, 128, kernel_size=3, stride=1, padding=1),
-        )
-        self.branch2 = nn.Sequential(
-            BasicConv3d(192, 16, kernel_size=1, stride=1),
-            SepConv3d(16, 32, kernel_size=3, stride=1, padding=1),
-        )
-        self.branch3 = nn.Sequential(
-            nn.MaxPool3d(kernel_size=(3, 3, 3), stride=1, padding=1),
-            BasicConv3d(192, 32, kernel_size=1, stride=1),
+class Branch0(nn.Sequential):
+    def __init__(self, in_planes, out_planes):
+        super(Branch0, self).__init__(
+            BasicConv3d(in_planes, out_planes, kernel_size=1, stride=1)
```

Comment:
TorchVision has a standard block for `BasicConv3d`. It's called `Conv3dNormActivation`.

`SepConv3d` can also be rewritten based on it. Not sure if we should call them separable conv because there are normalization and activation layers in between them but we can review naming later.